high_card.rb

```diff
@@ -1,5 +1,45 @@
-# Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
+require "pry"
+ #Use these two arrays to generate a deck of cards.
+ #capitalize so they are constant and won't change
+  RANKS = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"]
+  SUITS = [ "hearts", "spades", "clubs", "diamonds" ]
+  # '$' makes players a global variable
+  +@players = []
```

Comment:
You don't necessarily need to make this an instance variable here since we are not putting this in a class, It will work the same without using one!